main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: baby
+//  action 1: the baby cries
+baby.cry();
+//  action 2: the baby grabs
+baby.grab();
+//  action 3: the baby eats
+baby.eat();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: bird
+//  action 1: the bird flies
+bird.fly();
+//  action 2: the bird sleeps
+bird.sleep();
+//  action 3: the bird pecks
+bird.peck();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: bitwig
+//  action 1:  bitwig checks the contents of custom VST folder
+bitwig.customVstFolderContent();
+//  action 2: bitwig deactivates the audio engine upon CPU overload
+bitwig.deactivateAudioEngine();
+//  action 3: bitwig sleeps a VST when not in use
+bitwig.sleepVST();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: lightsaber
+//  action 1: the lightsaber cuts
+lightsaber.cut();
+//  action 2: the lightsaber deflects a blaster bolt
+lightsaber.blasterDeflect();
+//  action 3: the lightsaber changes to a random color
+lightsaber.randomColor();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: iphone
+//  action 1: the iphone takes a picture
+iphone.takePicture();
+//  action 2: the iphone sends a text
+iphone.sendText();
+//  action 3: the iphone updates its current GPS location
+iphone.gpsLocationUpdate();

```

Comment:
`update` is the verb, and it should be the first word in your pseudo-code function.
- [ ] revise